man.Rmd

```diff
@@ -583,15 +688,22 @@ There are two basic options:
     #' }
     ```
 
-[^man-7]: You used to be able to use `\donttest{}` for a similar purpose, but we no longer recommend it because CRAN sets a special flag that causes the code to be executed anyway.
+[^man-14]: You used to be able to use `\donttest{}` for a similar purpose, but we no longer recommend it because CRAN sets a special flag that causes the code to be executed anyway.
 
 We generally recommend using `try()` so that the reader can see an example of the error in action.
 
+::: callout-warning
+## Submitting to CRAN
+
+For the initial CRAN submission of your package, all functions must have at least one example and the example code can't all be wrapped inside `\dontrun{}`.
+If the code can only be run under specific conditions, use the techniques below to express those pre-conditions.
```

Comment:
I took above-average care with this wording, trying to really capture how to thread this needle.